DEVELOPERS.md

```diff
@@ -58,6 +58,14 @@ to accomplish this. To start the server:
 When asked, provide a valid user name (in the form of `user@domain`) and password so
 the application can start in the context of a logged in user.
 
+When the dev server is started, it will attempt to open a new browser window/tab on
+your system default browser to the app's running URL.  This behavior can be modified
+by specifying the `BROWSER` environment variable.  Possible values are:
+    BROWSER=none               # disable the feature
+    BROWSER=google-chrome      # open a new tab in chrome on Linux
+    BROWSER='google chrome'    # open a new tab in chrome on MaxOS
```

Comment:
Nit-pick: I think it would be nice also add version for windows:

`BROWSER=chrome`